docker-compose.yml

```diff
@@ -17,22 +17,23 @@ services:
 
   web:
     build:
-      context: ./backend
+      # context: ./backend
+      context: .
     container_name: fastapi-app
     ports:
       - "8000:8000"
     depends_on:
       - db
-    # environment:
-    #   - DATABASE_URL=postgresql://username:password@db/dbname
-    #   - CHOKIDAR_USEPOLLING=true
+    environment:
+      - PYTHONPATH=/app/backend
+      - CHOKIDAR_USEPOLLING=true
     env_file:
       - ./backend/.env  # or wherever your .env file is located
     networks:
       - app-network
     volumes:
-      - ./backend:/app
-
+      # - ./backend:/app
+      - .:/app
```

Comment:
What is the reasoning behind this? This would make the whole project get put into the docker container for the FastAPI backend.